### ISSUE TITLE
access reports

### ISSUE THREAD

**Issue by frbitten** [hoytech/strfry#103]
hi
Is there a way to have access data? Number of simultaneously connected clients, daily totals?

best regards

**Comment by bleetube** [comment]
Your webserver should be able to report these metrics

**Comment by hoytech** [comment]
Yep what @bleetube said: Easiest way currently is to throw nginx in front of strfry and process its logs.
